### PR TITLE
Add Katrina to the team page on Design System website

### PR DESCRIPTION
**Pull request by SaraC19**
Have updated the list of team members to include our new team member Katrina

### CODE REVIEW

File: src/design-system-team.md.njk [hunk 1]
```diff
@@ -23,6 +23,7 @@ If you want to contact the team you can
 - Imran Hussain – Community Manager
 - Izabela Podralska – Delivery Manager
 - Joe Lanman – Lead Interaction Designer
+- Katrina Birch - User Researcher
```
Comment (EoinShaughnessy) [inline src/design-system-team.md.njk:26]:
```suggestion
- Katrina Birch – User Researcher
```

Comment (EoinShaughnessy) [reply src/design-system-team.md.njk:26]: Suggesting this change as we insert dashes between the name and the person's role.

Comment (vanitabarrett) [reply src/design-system-team.md.njk:26]: @SaraC19 Eoin's comment here is about making sure we use the right kind of dash to be consistent. You should be able to press 'Commit suggestion' and 'Commit changes' to accept this suggestion - are you ok to do that? Happy to walk you through it.

Comment (SaraC19) [reply src/design-system-team.md.njk:26]: Ah ha - thank you! I'll press some buttons now and will see if it works.

**Comment by EoinShaughnessy** [comment]
Looks great, thanks @SaraC19!

**Pull request merged**